### PR TITLE
seamless update

### PR DESCRIPTION
**Pull request by KennethKnows**
Enhance

### CODE REVIEW

File: src/assets/img/1.png
Comment (kynttt) [inline src/assets/img/1.png:1]: delete this png file.

**Review by kynttt — CHANGES_REQUESTED** [review]
check comment

File: src/assets/img/1.png
Comment (KennethKnows) [reply src/assets/img/1.png:1]: Deleted the image.

**Review by kynttt — APPROVED** [review]
approved.

**Pull request merged**